pkg/targetgroupbinding/targets_manager.go

```diff
@@ -0,0 +1,276 @@
+package targetgroupbinding
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	elbv2sdk "github.com/aws/aws-sdk-go/service/elbv2"
+	"k8s.io/apimachinery/pkg/util/cache"
+	"sigs.k8s.io/aws-alb-ingress-controller/pkg/aws/services"
+	"sync"
+	"time"
+)
+
+const (
+	defaultTargetsCacheTTL            = 10 * time.Minute
+	defaultRegisterTargetsChunkSize   = 200
```

Comment:
How did you determine this? Is it based on ELBv2 documentation? does the register and deregister chunk size have to be the same?

Reply:
it's based on internal communication with elbv2 team. register/deregister batch size theoretically can be different.